src/views/add_event.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="">
+  <form action="/insertEvent" method="post">
+      <label for="title-event">Title:</label>
+      <input type="text" name="title" class="pa2 bg-light-gray" placeholder="Name of the event?" id="title-event" required>
+
+      <label for="day-input" class="">Day:</label>
+      <input type="number" name="day" min="1" max="31" value="1" class="bg-light-gray" id="day-input" required>
+
+      <label for="month-input" class="">Month:</label>
+      <input type="number" name="month" min="1" max="12" value="1" class="bg-light-gray" id="month-input" required>
+
+      <label for="year-input" class="">Year:</label>
```

Comment:
I know you hate this comment - could we remove all these ":" ? :angel:

Reply:
:+1: